firefox_ios/views/unified_metrics.view.lkml

```diff
@@ -0,0 +1,738 @@
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Bookmarks Open"
+  }
+
+  dimension: metrics__labeled_counter__bookmarks_view_list {
+    hidden: yes
+    sql: ${TABLE}.metrics.labeled_counter.bookmarks_view_list ;;
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Bookmarks View List"
+  }
+
+  dimension: metrics__labeled_counter__reading_list_add {
+    hidden: yes
+    sql: ${TABLE}.metrics.labeled_counter.reading_list_add ;;
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Reading List Add"
+  }
+
+  dimension: metrics__labeled_counter__reading_list_delete {
+    hidden: yes
+    sql: ${TABLE}.metrics.labeled_counter.reading_list_delete ;;
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Reading List Delete"
+  }
+
+  dimension: metrics__labeled_counter__search_counts {
+    hidden: yes
+    sql: ${TABLE}.metrics.labeled_counter.search_counts ;;
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Search Counts"
+  }
+
+  dimension: metrics__labeled_counter__tabs_close {
+    hidden: yes
+    sql: ${TABLE}.metrics.labeled_counter.tabs_close ;;
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Tabs Close"
+  }
+
+  dimension: metrics__labeled_counter__tabs_open {
+    hidden: yes
+    sql: ${TABLE}.metrics.labeled_counter.tabs_open ;;
+    group_label: "Metrics Labeled Counter"
+    group_item_label: "Tabs Open"
+  }
+
+  dimension: metrics__string__preferences_mail_client {
+    type: string
+    sql: ${TABLE}.metrics.string.preferences_mail_client ;;
+    group_label: "Metrics String"
+    group_item_label: "Preferences Mail Client"
+  }
+
+  dimension: metrics__string__preferences_new_tab_experience {
+    type: string
+    sql: ${TABLE}.metrics.string.preferences_new_tab_experience ;;
+    group_label: "Metrics String"
+    group_item_label: "Preferences New Tab Experience"
+  }
+
+  dimension: metrics__string__search_default_engine {
+    type: string
+    sql: ${TABLE}.metrics.string.search_default_engine ;;
+    group_label: "Metrics String"
+    group_item_label: "Search Default Engine"
+  }
+
+}
+
+view: unified_metrics__ping_info__experiments {
+  dimension: key {
+    type: string
+    sql: ${TABLE}.key ;;
+  }
+
+  dimension: value__branch {
+    type: string
+    sql: ${TABLE}.value.branch ;;
+    group_label: "Value"
+    group_item_label: "Branch"
+  }
+
+  dimension: value__extra__type {
+    type: string
+    sql: ${TABLE}.value.extra.type ;;
+    group_label: "Value Extra"
+    group_item_label: "Type"
+  }
+}
+
+view: unified_metrics__metrics__labeled_counter__tabs_open {
+  dimension: key {
+    type: string
+    sql: ${TABLE}.key ;;
+  }
+
+  dimension: value {
+    type: number
+    sql: ${TABLE}.value ;;
+  }
+
+  measure: counts {
+    type:  sum
+    sql:  ${value} ;;
+  }
+}
+
+view: unified_metrics__metrics__labeled_counter__tabs_close {
+  dimension: key {
```

Comment:
Unfortunately the filter suggestions don't work for these joined views. Basically looked tries to query the table `unified_metrics__metrics__labeled_counter__tabs_close` directly, which of course doesn't exist. We have two options: disable filters suggestions for these fields, or create another explore and [use that to load the suggestions for all of these](https://docs.looker.com/data-modeling/learning-lookml/filter-suggestions#suggest_dimension_(field)_and_suggest_explore_(field)).

Reply:
I'm not sure how another explore would solve this problem without having one explore per unnested field. I don't think there's any harm in having having an empty filter suggest here?

Reply:
It is a sub-par user experience to leave them empty, but I'm okay adding this as a future improvement if it's too much to ask right now.